### PR TITLE
livenessProbe TestDBConnection code completion.

### PR DESCRIPTION
**Pull request by msvinaykumar**

### CODE REVIEW

File: src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.hibernate.SessionFactory;
+
+import java.util.List;
+
+public class TestDBConnection {
+    public static void main(String[] args) {
+        System.out.println("Checking DB connection...");
+        SessionFactory factory = KruizeHibernateUtil.getSessionFactory();
+        Session session = factory.openSession();
+        String sql = "SELECT routine_name FROM information_schema.routines WHERE routine_schema='public' AND routine_type='FUNCTION'";
```
Comment (dinogun) [inline src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java:29]: Have this in a constants file

File: src/main/java/com/autotune/dbactivites/init/KruizeHibernateUtil.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+public class KruizeHibernateUtil {
+    private static final SessionFactory sessionFactory;
+    private static final Logger LOGGER = LoggerFactory.getLogger(KruizeHibernateUtil.class);
+
+    static {
+        SessionFactory sfTemp = null;
+        try {
+            Configuration configuration = new Configuration().configure();
+            String connectionURL = "jdbc:postgresql://" +
```
Comment (dinogun) [inline src/main/java/com/autotune/dbactivites/init/KruizeHibernateUtil.java:32]: Push this to a constants file

File: src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import com.autotune.dbactivites.init.KruizeHibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.util.List;
+
+public class TestDBConnection {
+    public static void main(String[] args) {
+        System.out.println("Checking DB connection...");
```
Comment (dinogun) [inline src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java:26]: Will this be converted to a LOGGER later?

File: src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        SessionFactory factory = KruizeHibernateUtil.getSessionFactory();
+        Session session = factory.openSession();
```
Comment (chandrams) [inline src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java:28]: Shouldn't we add a null check here ? Should this code be within try/catch ?

Comment (msvinaykumar) [reply src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java:30]: The purpose of this script is solely to facilitate the Liviliness probe, which aims to detect system errors that may cause the Kruize pod to fail to start if there is a problem with the database connection. Therefore, the script intentionally avoids using a try-catch mechanism.

**Comment by dinogun** [comment]
@msvinaykumar please fix the conflicts

**Comment by msvinaykumar** [comment]
> @msvinaykumar please fix the conflicts

done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**